lib/spack/spack/test/concretize_preferences.py

```diff
@@ -15,6 +17,9 @@
 from spack.spec import Spec
 from spack.version import Version
 
+pytestmark = pytest.mark.skipif(sys.platform == "win32",
+                                reason="does not run on windows")
+
```

Comment:
We might want to investigate why this is failing on Windows a bit more - I think we'll want to be able to set concretization preferences starting out.

Reply:
Handled.